vmm/sandbox/src/container/handler/spec.rs

```diff
@@ -71,6 +77,20 @@ where
         }
         // Update sandbox files mounts for container
         container_mounts(&shared_path, spec);
+
+        // Update ipc and uts namespace for container
+        if let Some(linux) = spec.linux.as_mut() {
+            for ns in linux.namespaces.iter_mut() {
+                if ns.r#type == IPC_NAMESPACE || ns.r#type == UTS_NAMESPACE {
+                    ns.path = format!("{}/{}", SANDBOX_NS_PATH, ns.r#type);
+                    continue;
+                }
+            }
+        }
+
+        // Update container hostname to sandbox hostname
+        spec.hostname = hostname;
```

Comment:
will these two values be different？

Reply:
Containerd would not set hostname for container, it only set it to pod.